### PR TITLE
feat: add addresses tab, create and edit addresses, addresses list

### PR DESCRIPTION
**Pull request by Dmytro233**
Description:

- add new tab with address book
- create and edit addresses
- list of addresses
link - https://elasticpath.atlassian.net/browse/PB-9042.

Create a new folder src/containers/b2b/AccountDetailsPage. There is a main page file AccountDetailsPage.tsx with tabs files:
AccountDetailsPage > 
       - AccountDetailsPage.tsx
        - Overviews.tsx
        - AddressBook.tsx
        - PaymentInstruments.tsx

### CODE REVIEW

File: src/ep.config.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "cortexApi": {
     "path": "/cortex",
-    "scope": "vestri",
-    "pathForProxy": "http://localhost:9080",
+    "scope": "mobee",
```
Comment (shaunmaharaj) [inline src/ep.config.json:4]: You need to revert the changes in this file so they don't get merged. The local config shouldn't be pushed

Comment (Dmytro233) [reply src/ep.config.json:4]: ok. forgot this) I will revert the changes

File: src/components/src/ProfileAddressesWithDefaults/profileaddresses.main.tsx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    );
+  }
+
+  renderAddress(addressElement) {
+    const { onEditAddress, chosenBillingUri, chosenShippingUri } = this.props;
+    const { name, address, self } = addressElement;
+    let defaultAddress = '';
+
+    if (self.uri === chosenBillingUri && self.uri === chosenShippingUri) {
+      defaultAddress = 'Default Bill/Ship To';
```
Comment (shaunmaharaj) [inline src/components/src/ProfileAddressesWithDefaults/profileaddresses.main.tsx:142]: These strings should be externalized in our localization files

**Comment by shaunmaharaj** [comment]
Thanks for this @Dmytro233! I left a couple comments and tested. We can merge once those are resolved.
Everything looks great! I like how clean you kept your code here too 👍

**Review by petro97 — APPROVED** [review]

**Comment by shaunmaharaj** [comment]
Thanks @Dmytro233! Merging now
